firebase-crashlytics/src/androidTest/java/com/google/firebase/crashlytics/internal/persistence/CrashlyticsReportPersistenceTest.java

```diff
@@ -199,7 +232,10 @@ public void testFinalizeReports_capsReports() throws IOException {
     for (int i = 0; i < 10; i++) {
       persistReportWithEvent(reportPersistence, "testSession" + i, true);
     }
-    reportPersistence.finalizeReports("skippedSession");
+
+    final long endedAt = System.currentTimeMillis();
+
+    reportPersistence.finalizeReports("skippedSession", endedAt);
```

Comment:
if it is not relevant to the test, I'm happy with inlining, e.g., 0L for all of these

Reply:
👍